data/en/entityloadbypk.json

```diff
@@ -1,18 +1,18 @@
 {
 	"name":"entityLoadByPK",
 	"type":"function",
-	"syntax":"entityLoadByPK(entityName, id)",
+	"syntax":"entityLoadByPK(entity, id)",
 	"returns":"any",
 	"related":["entityLoadByExample"],
-	"description":" Loads and returns an array of objects for given primary key.",
+	"description":"Loads and returns an array of objects for given primary key.\bUse this function to avoid specifying the required boolean parameter in `EntityLoad()`.",
```

Comment:
Is the `\b` intentional?

Reply:
Supposed to be a new line \n. Apparently my fingers were wider and faster than I intended. 

Fixed.